Error building with 1.4.1

My build process threw this error with bulkify `1.4.1`:
```
module.js:338
       throw err;
       ^
       Error: Cannot find module 'concat-stream'
       at Function.Module._resolveFilename (module.js:336:15)
       at Function.Module._load (module.js:278:25)
       at Module.require (module.js:365:17)
       at require (module.js:384:17)
       at Object.<anonymous> (/tmp/build/node_modules/bulkify/index.js:5:14)
       at Module._compile (module.js:460:26)
       at Object.Module._extensions..js (module.js:478:10)
       at Module.load (module.js:355:32)
       at Function.Module._load (module.js:310:12)
       at Module.require (module.js:365:17)
       at require (module.js:384:17)
       at nr (/tmp/build/node_modules/browserify/node_modules/module-deps/index.js:289:21)
       at /tmp/build/node_modules/browserify/node_modules/resolve/lib/async.js:44:21
       at ondir (/tmp/build/node_modules/browserify/node_modules/resolve/lib/async.js:187:31)
       at /tmp/build/node_modules/browserify/node_modules/resolve/lib/async.js:153:39
       at onex (/tmp/build/node_modules/browserify/node_modules/resolve/lib/async.js:93:22)
       at /tmp/build/node_modules/browserify/node_modules/resolve/lib/async.js:24:18
       at FSReqWrap.oncomplete (fs.js:95:15)
```
The issue seemed to be resolved when I downgraded to `1.4.0`.

--- a/package.json
+++ b/package.json
@@ -5,11 +5,11 @@
   "main": "index.js",
   "dependencies": {
     "bulk-require": "^1.0.0",
+    "concat-stream": "^1.4.5",
     "static-module": "^1.1.2",
     "through2": "~0.4.1"
   },
   "devDependencies": {
-    "concat-stream": "^1.4.5",
     "tape": "^2.12.1"
   },
   "scripts": {

